src/resources/enemy.py

```diff
@@ -0,0 +1,28 @@
+from random import randint
+
+from .AbstractDungeonEntity import AbstractDungeonEntity
+from .level import Level
+
+
+class Enemy(AbstractDungeonEntity):
+    """Enemy entity and hostile to players"""
+
+    def __init__(self, current_level: Level, symbol: str = '^') -> None:
+        super().__init__(
+            ground_symbol=current_level.board[
+                (x := randint(2, current_level.width - 2))
+            ][(y := randint(2, current_level.height - 2))],
+            x=x,
+            y=y,
+            symbol=symbol
+        )
+        self.current_level = current_level
+
+    def draw(self) -> None:
+        """Draws enemies to board"""
+        # while loop to ensure that enemies don't get placed in the same area
+        while str(self.current_level.board[self.y][self.x]) in (self.symbol, '$'):
+            self.x = randint(2, self.current_level.width - 2)
+            self.y = randint(2, self.current_level.width - 2)
+
+        self.current_level.board[self.y][self.x] = self.symbol
```

Comment:
Draw should only place a single enemy on the board where ever it's x and y position is.

Reply:
Weren't we having multiple enemies on the board? with a cap size of 10

Reply:
Yes, but that is not what draw does. An enemies draw only draws a single enemy. Every enemy entity will have its own draw.

Reply:
for now start with a single enemy so you can get its movement working more easilly.

Reply:
Ah alright

Reply:
Draw already only places a single enemy on the board though, I just have a for loop in the ``app.py`` to place all the enemies